tests/unit/all.ts

```diff
@@ -1,3 +1,4 @@
+import '@dojo/test-extras/support/loadJsdom';
```

Comment:
you might want to consider adding this to the `suites` array in the intern configuration...  I think that is where we will try to standardise on its location.